Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+pipeline {
+    agent {
+        label 'kie-rhel7'
+    }
+    tools {
+        maven 'kie-maven-3.5.4'
+        jdk 'kie-jdk1.8'
```

Comment:
Any chance we could run JDK 8 *and* JDK 11?
Not saying it's necessary, I'm just curious if it would be possible.